components/home-page/videos/Videos.module.css

```diff
@@ -7,19 +7,36 @@
     border: #a4a4a4 solid 1px !important;
 }
 
-.videoImage img:hover {
+/* .videoImage img:hover {
     border: 2px solid #1eb5ae !important
+} */
+.videoImage img:hover {
+    border: 2px solid #1eb5ae !important;box-shadow: 0px 0px 30px 0 rgba(0, 0, 0, 0.2) !important;
```

Comment:
Can you put each style in separate lines instead of in one line to improve the code readability.

Reply:
I updated the code, Thank you